packages/dds/merge-tree/src/test/client.applyMsg.spec.ts

```diff
@@ -1015,6 +1015,48 @@ describe("client.applyMsg", () => {
 
 			logger.validate({ baseText: "BBBBBB B" });
 		});
+		// TODO: see AB#29398
+		it.skip("obliterate with mismatched final states", () => {
+			const clients = createClientsAtInitialState(
+				{
+					initialState: "B{6666666}BBB{666666}BB",
```

Comment:
some i've found useful is to change these all to value which are not reused in the test below, so from the results it is easier to see what has change vs not. i usually use numbers that correspond to the positions, but now that fields have numbers maybe something different would be better. i'd probably change this to something like
```suggestion
					initialState: "0{zzzzzzz}123{yyyyyy}45",
```